conf/test_taxonomy.config

```diff
@@ -26,6 +26,7 @@ params {
 
     run_taxa_classification = true
     annotation_tool         = 'pyrodigal'
+    save_annotations        = true
```

Comment:
Will add those to the `test_bgc.config`, keep them here as well.

Reply:
I left it as is. Why? Of the test profiles that are used by GHA we should have 1 with and 1 without the contig length params. Since the BGC test workflow uses the same samplesheet but fewer tools (and no tax workflow), I thought to save more resources this way around.
Note: I also added the contig length params to the `full_test.config`